docker-compose.yml

```diff
@@ -1,19 +1,55 @@
-version: '3'
+version: '3.4'
 services:
   truffle:
     build:
       context: .
-    image: market-protocol:1.0.0
+    image: market-protocol:1.0.1
     command: truffle develop
+    tmpfs: 
+      - /app/build
+      - /app/node_modules:exec,mode=777
     volumes:
-      - ./:/app
+      - type: volume
+        source: ./
+        target: /app
+        name: truffle_volume
     tty: true
     stdin_open: true
 
   eth-bridge:
-    image: perfectmak/ethereum-bridge:0.5.5
+    image: cryptomental/ethereum-bridge-coverage:0.5.6
```

Comment:
@cryptomental - just to be sure - If I fork the ethereum bridge you modified into the MarketProtocol repo and change this line, we should be okay?

Reply:
The changes are small, see here https://github.com/perfectmak/ethereum-bridge/compare/master...cryptomental:master

After the PR we could merge the changes into MarketProtocol's ethereum bridge and push the docker image to docker hub from there, to have marketprotocol/ethereum-bridge in that line. Or try to push this upstream into the original eth bridge, but I am not sure that they will accept the increased gas limit change.